letsencrypt/defaults.yaml

```diff
@@ -6,6 +6,9 @@ letsencrypt:
     - python-certbot-apache
   service: certbot.timer
   cli_install_dir: /opt/letsencrypt
+  config_permissions:
+    user: root
+    group: root
```

Comment:
@pprkut, what do you think of creating

```
config_dir:
  path: /etc/letsencrypt
  user: root
  group: root
  mode: 755
```

and then use those vars in the `letsencrypt-config-directory` and `letsencrypt-config` states? That way, it would allow to set these vars to other paths if desired?

Reply:
Sounds good! I'll have a look at it

Reply:
@pprkut, had any time to check this, so we can merge?

Reply:
I had now :)
Is this more or less what you had in mind?